### PR TITLE
Update Readme.md with roadmap.

### PR DESCRIPTION
**Pull request by arijitAD**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -38,3 +38,25 @@ curl --include \
    --header "Sec-WebSocket-Version: 13" \
    http://localhost:8888/ws
 ```
+
+## Roadmap
+
+- [ ] Framing
+  - [x] Text Messages
+  - [ ] Binary Messages
```
Comment (dryajov) [inline Readme.md:46]: What is the issue with binary messages?

Comment (arijitAD) [reply Readme.md:46]: Implemented it here. https://github.com/status-im/nim-ws/pull/9/commits/9e98713be0952a129ac5403017176cee18d9cf11
Just had to expose the opcode in `recv`

**Comment by arijitAD** [comment]
@dryajov Binary message test is passing. https://status-im.github.io/nim-ws/
Updated the roadmap.

File: Readme.md [hunk 2]
```diff
@@ -38,3 +38,26 @@ curl --include \
    --header "Sec-WebSocket-Version: 13" \
    http://localhost:8888/ws
 ```
+
+## Roadmap
+
+- [ ] Framing
```
Comment (dryajov) [inline Readme.md:44]: As i understand it, i think framing is supported to the extent that the RFC specifies it, however any extension can have it's own framing.

File: Readme.md (hunk 1 above)
Comment (jangko) [reply Readme.md]: ok. thx

File: Readme.md (hunk 2 above)
Comment (jangko) [reply Readme.md:44]: ok. thx

**Pull request merged**